app/volume.go

```diff
@@ -41,6 +47,10 @@ func SnapshotCmd() cli.Command {
 				Name:  FlagManagerURL,
 				Usage: "Longhorn manager API URL",
 			},
+			cli.StringFlag{
+				Name:  FlagCurrentNode,
```

Comment:
Why do we need this?

Reply:
OK I see. But what if the specified node is down? Seems better if we randomly choose a ready node.

Reply:
Thank you. Fixed